### ISSUE TITLE
Practical positioning examples are not accessible

### ISSUE THREAD

**Issue by pepelsbey** [mdn/content#30473]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Practical_positioning_examples

### What information was incorrect, unhelpful, or incomplete?

While examples used in the guide are accessible via keyboard, they are not accessible to screen reader users.

- [Hidden info panel](https://mdn.github.io/learning-area/css/css-layout/practical-positioning-examples/hidden-info-panel.html) (uses links instead of buttons)
- [Tabbed info box](https://mdn.github.io/learning-area/css/css-layout/practical-positioning-examples/info-box.html) (uses checkbox instead of button)

For such patterns to work properly, it’s required to manage focus and use ARIA roles. For example:

- [APG: Tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)
- [APG: Menu Button](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/)

I think it’s important to have accessible examples in our guides.

### What did you expect to see?

Examples are fully accessible from the keyboard and to screen readers.

### Do you have anything more you want to share?

There are two ways of solving this, as far as I see:

- Use simpler examples that don’t require interactive elements.
- Fix existing examples to be fully accessible (it might complicate examples).

**Comment by Josh-Cena** [comment]
This is a duplicate of https://github.com/mdn/content/issues/24806. This one points out one more inaccessible example; I'll add it to the other issue since that one contains more information and discussion.
